### DISCUSSION TITLE
How to get current dot's directory

### DISCUSSION THREAD

**Discussion by AdamYellen** [volllly/rotz#111] [Q&A] [answered]
If I wanted to execute a custom install script that is stored in the dot's directory, how can I find it?

**Comment by volllly** [comment, marked as answer]
Hey, you could do it like that:

```yaml
installs: "{{ config.dotfiles }}/{{ name }}/install.ps1"
```

`config.dotfiles` is the dotfiles repo path (or the default path if no path is set in the config file).

`name` is the name of the do (including subdirectories if its nested)

**Reply by AdamYellen** [reply]
Thank you!
